libraries/AP_MSP/AP_MSP_Telem_Backend.cpp

```diff
@@ -1057,6 +1057,22 @@ void AP_MSP_Telem_Backend::hide_osd_items(void)
             BIT_SET(osd_hidden_items_bitmask, OSD_RTC_DATETIME);
         }
     }
-}
+} 
 
+bool AP_MSP_Telem_Backend::osd_stats_enabled() const
```

Comment:
Is this really
```suggestion
bool AP_MSP_Telem_Backend::displaying_statistics_screen() const
```

?

Reply:
yes and no, there's no statistics screen, there are screens where you can enable the stats OSD panel mixed with whatever the user wants, in this PR it's the same, you can have the stats iten mixed with other items even on OSD screen one, it is true though that this hack overrides telemetry with min/max values so I guess there's no point in mixing them. the method should probably have a name like osd_statistics_panel_enabled() because that is what I actually check

Reply:
but in the end what it really means is "displaying_stats_screen()" :-(